{
  "affected_release": null,
  "package_state": [
    {
      "product_name": "Red Hat Enterprise Linux 6",
      "fix_state": "Not affected",
      "package_name": "libssh2",
      "cpe": "cpe:/o:redhat:enterprise_linux:6"
    },
    {
      "product_name": "Red Hat Enterprise Linux 7",
      "fix_state": "Not affected",
      "package_name": "libssh2",
      "cpe": "cpe:/o:redhat:enterprise_linux:7"
    },
    {
      "product_name": "Red Hat Enterprise Linux 8",
      "fix_state": "Affected",
      "package_name": "libssh",
      "cpe": "cpe:/o:redhat:enterprise_linux:8"
    },
    {
      "product_name": "Red Hat Enterprise Linux 8",
      "fix_state": "Not affected",
      "package_name": "libssh2",
      "cpe": "cpe:/o:redhat:enterprise_linux:8"
    },
    {
      "product_name": "Red Hat Virtualization 4",
      "fix_state": "Under investigation",
      "package_name": "libssh",
      "cpe": "cpe:/:redhat:enterprise_linux:::hypervisor"
    }
  ],
  "threat_severity": "Low",
  "public_date": "2019-12-05T00:00:00Z",
  "bugzilla": {
    "description": "\nCVE-2019-14889 libssh: unsanitized location in scp could lead to unwanted command execution\n    ",
    "id": "1772523",
    "url": "https://bugzilla.redhat.com/show_bug.cgi?id=1772523"
  },
  "cvss": {
    "cvss_base_score": "",
    "cvss_scoring_vector": "",
    "status": ""
  },
  "cvss3": {
    "cvss3_base_score": "7.1",
    "cvss3_scoring_vector": "CVSS:3.0/AV:N/AC:H/PR:L/UI:R/S:U/C:H/I:H/A:H",
    "status": "draft"
  },
  "iava": "",
  "cwe": "CWE-77",
  "statement": "",
  "acknowledgement": "\nRed Hat would like to thank libssh project for reporting this issue. Upstream acknowledges Cure53 as the original reporter.\n    ",
  "name": "CVE-2019-14889",
  "document_distribution": "",
  "details": [
    "\nNo description is available for this CVE.\n    "
  ],
  "references": [
    "\nhttps://www.libssh.org/security/advisories/CVE-2019-14889.txt\n    "
  ]
}